server/src/test/scala/io/delta/sharing/server/DeltaSharingServiceSuite.scala

```diff
@@ -38,7 +39,21 @@ import io.delta.sharing.server.util.JsonUtils
 class DeltaSharingServiceSuite extends FunSuite with BeforeAndAfterAll {
 
   def shouldRunIntegrationTest: Boolean = {
-    sys.env.get("AWS_ACCESS_KEY_ID").exists(_.length > 0)
+    sys.env.get("AWS_ACCESS_KEY_ID").exists(_.length > 0) &&
+      sys.env.get("AZURE_TEST_ACCOUNT_KEY").exists(_.length > 0) &&
+      sys.env.get("GOOGLE_APPLICATION_CREDENTIALS").exists(_.length > 0)
+  }
+
+  private def buildGoogleServiceAccountKeyFile: Unit = {
+    // Only write tmp file if the key content is defined, otherwise the key is loaded directly from
+    // the key file path in GOOGLE_APPLICATION_CREDENTIALS.
+    if (sys.env.get("GOOGLE_SERVICE_ACCOUNT_KEY").exists(_.length > 0)) {
+      val serviceAccountKey = sys.env("GOOGLE_SERVICE_ACCOUNT_KEY")
+      val fileWriter = new FileWriter(new File("/tmp/google_service_account_key.json"))
+      fileWriter.write(serviceAccountKey)
+      fileWriter.flush()
+      fileWriter.close()
```

Comment:
We can use [FileUtils.writeStringToFile](https://github.com/delta-io/delta-sharing/blob/7dbf108f7cd2f866392d461f743d82144f9d4c17/server/src/test/scala/io/delta/sharing/server/config/ServerConfigSuite.scala#L32) to save multiple lines.